### DISCUSSION TITLE
Make the Blue Humanoid Sprite you get when click and drag your body change shape depending on what is below it

### DISCUSSION THREAD

**Discussion by Akbadain29** [unitystation/unitystation#8957] [Ideas]
## Description
It would be a good addition if when you click dragged the blue humanoid sprite it changes depending what it was on top of, which then clearly communicates to the player that it will perform a particular function like buckling a player to a chair. It would help new players know that the action is going to be performed when they let go of the mouse button.

**Comment by corp-0** [comment]
I think this should be doable by using a shader instead of an actual sprite we have in our assets. So the shader would take the appearance of the game object your dragging and it would work with anything.

This is a good idea, +1 from me.
